components/server/src/util/request-context.ts

```diff
@@ -165,7 +166,8 @@ export type RequestContextSeed = Omit<RequestContext, "requestId" | "startTime"
 };
 
 /**
- * The context all our request-handling code should run in.
+ * Creates a _root_ context request-handling code should run in. MANDATORY for any authorization to work.
```

Comment:
if we never want to nest these, we should add a check and throw an error in case we do